components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/java/org/wso2/carbon/identity/api/server/application/management/v1/core/ServerApplicationManagementService.java

```diff
@@ -1139,6 +1150,37 @@ private boolean isAllowUpdateSystemApplication(String appName, ApplicationPatchM
         return false;
     }
 
+    /**
+     * Check whether the updating system application's Inbound OAuth Configs are allowed or not.
+     *
+     * @param appId Application id.
+     * @param newOidcConfigModel New oidc config model.
+     * @return true if allowed.
+     */
+    private boolean isAllowUpdateSystemApplicationInboundOAuthConfigs(String appId,
```

Comment:
method name is too long. can you come up with shorter versions? :)

Reply:
Addressed with https://github.com/wso2/identity-api-server/pull/548/commits/81a64576439cc8ae2be2a170950e013e2fa4bb77